misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/LineScreen.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+
+import PickLineScreen from './picking/PickLineScreen';
+import DistributionLineScreen from './distribution/DistributionLineScreen';
+
+import { selectWFProcessFromState } from '../../reducers/wfProcesses_status';
+
+const getLineComponent = (appId) => {
+  switch (appId) {
+    case 'picking':
+      return PickLineScreen;
+    case 'distribution':
+      return DistributionLineScreen;
+    default:
+      return null;
+  }
+};
+
+class LineScreen extends PureComponent {
+  render() {
+    const { appId } = this.props;
+    const Component = getLineComponent(appId);
```

Comment:
use something with more meaning like `LineItem` .  Something without `Component` in the name as we know already it is a component